### ISSUE TITLE
Catalog of functions is points to a 404

### ISSUE THREAD

**Issue by kgilds** [sfirke/janitor#180]
## Bug reports

Catalog of functions in readme is not working. I updated the readme.md file but probably should have updated the Rmd. file.   

.

---

Brief description of the problem

```r
# insert reprex here
```

**Comment by sfirke** [comment]
Good catch. Because of the way packagedown works I renamed the vignette
from introduction to janitor. I forgot to update the link in that readme
file.

You're right that it should be changed in the .Rmd file. If you want to
submit a PR that does that, go for it :)

On Feb 10, 2018 8:45 PM, "Kevin Gilds" <notifications@github.com> wrote:

Bug reports

Catalog of functions in readme is not working. I updated the readme.md file
but probably should have updated the Rmd. file.

.
------------------------------

Brief description of the problem

# insert reprex here

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/sfirke/janitor/issues/180>, or mute the thread
